.github/workflows/cucumber-tests.yml

```diff
@@ -2,7 +2,7 @@ name: Cucumber Tests
 
 on:
   push:
-    branches: [master]
+    branches: [master, qa-updated-endpoints]
```

Comment:
Does it still need qa-updated-endpoints?

Reply:
unless he's continually pushing changes here for future updates? I wouldn't be angry at this.

Reply:
removed the branch from the flow.